torch/_dynamo/output_graph.py

```diff
@@ -778,6 +778,10 @@ def create_proxy(
         if nn_module_stack:
             rv.node.meta["nn_module_stack"] = nn_module_stack.copy()
 
+        torch_api_call = tx.torch_api_call
+        if torch_api_call:
+            rv.node.meta["torch_api_call"] = torch_api_call
```

Comment:
Need a better name for this. I don't want this only to be used for torch.* function. I suspect we will need to extend this for other buildin.functions and tensor.methods(). 

alternative names: original_fn, source_fn

Reply:
makes sense, we want to have things like `operator.add` here as well

Reply:
Update to source_fn.

Reply:
btw is `operator.add` supported now?

Reply:
I simplified the implementation. We can directly store the original node.target in node.meta["source_fn"], and propagate it along lowering. 

See updated summary. 

This now also cover the case for operator.add.